pkg/accessor/events/events_accessor.go

```diff
@@ -0,0 +1,93 @@
+package events
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eventbridge"
+	"github.com/aws/aws-sdk-go/service/eventbridge/eventbridgeiface"
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+)
+
+type eventsAccessor struct {
```

Comment:
Reading this code, I think a lot of it could be collapsed -- happy to work on it w/ ya.